src/utils.py

```diff
@@ -47,10 +50,43 @@ def safe_get_file(filepath: str) -> Optional[List[str]]:
     return content
 
 
-def generate_password():
+def generate_password() -> str:
     """Creates randomized string for use as app passwords.
 
     Returns:
         String of 32 randomized letter+digit characters
     """
     return "".join([secrets.choice(string.ascii_letters + string.digits) for _ in range(32)])
+
+
+def map_env(env: list[str]) -> dict[str, str]:
+    """Builds environment map for arbitrary env-var strings.
+
+    Returns:
+        Dict of env-var and value
+    """
+    map_env = {}
+    for var in env:
+        key = "".join(var.split("=", maxsplit=1)[0])
+        value = "".join(var.split("=", maxsplit=1)[1:])
+
+        map_env[key] = value
+
+    return map_env
```

Comment:
**niptick** one-liner with walrus, since I'm a big fan. Up to you if you like this

```python
return {parts[0]: "".join(parts[1:]) for var in env if (parts := var.split("=", maxsplit=1))}
```